engines/router/missionctl/server/application.go

```diff
@@ -47,37 +48,48 @@ func Run() {
 	// Init Sentry, defer closing client
 	defer initSentryClient(cfg)()
 
-	// Init mission control
-	missionCtl, err := missionctl.NewMissionControl(
-		nil,
-		cfg.EnrichmentConfig,
-		cfg.RouterConfig,
-		cfg.EnsemblerConfig,
-		cfg.AppConfig,
-	)
-	if err != nil {
-		log.Glob().Panicf("Failed initializing Mission Control: %v", err)
-	}
-
-	// Register handlers
-	http.Handle("/v1/internal/", http.StripPrefix(
-		"/v1/internal",
-		handlers.NewInternalAPIHandler([]string{
-			cfg.EnsemblerConfig.Endpoint,
-			cfg.EnrichmentConfig.Endpoint,
-		}),
-	))
-	http.Handle("/v1/predict", sentry.Recoverer(handlers.NewHTTPHandler(missionCtl)))
-	http.Handle("/v1/batch_predict", sentry.Recoverer(handlers.NewBatchHTTPHandler(missionCtl)))
-	// Register metrics handler
-	if cfg.AppConfig.CustomMetrics {
-		http.Handle("/metrics", promhttp.Handler())
-	}
-
-	// Serve
-	log.Glob().Infof("listening at port %d", cfg.Port)
-	if err := http.ListenAndServe(cfg.ListenAddress(), http.DefaultServeMux); err != nil {
-		log.Glob().Errorf("Failed to start Turing Mission Control API: %s", err)
+	if cfg.RouterConfig.Protocol == config.UPI {
```

Comment:
hmmm

I will remove the `default http` and change it into `required` in `config.go` - this will enforce the protocol is always passed in. 

Will also do what u suggestion, to test again no matches.

Reply:
just realized i cant put it as required, until i patch the API Server. will do so in an upcoming MR. setting it back as default for now.